arguments_parsing.py

```diff
@@ -14,10 +19,10 @@ def parse_connection_arguments():
     connection_parameters = {}
     config_file_path = f"{CONFIG_PATH}/{CONNECTION_CONFIG_NAME}"
     if not os.path.isfile(config_file_path):
-        print(f"Error, there is no file {config_file_path}")
+        logger.critical(f"Error, there is no file {config_file_path}")
         raise Exception(f"Error, there is no file {config_file_path}")
 
-    with open() as connection_config_file:
+    with open(config_file_path) as connection_config_file:
         json_parameters = json.load(connection_config_file)
         if not set(['dbname', 'host_ip', 'user', 'port']).issubset(set(json_parameters.keys())):
             raise RuntimeError("There are not all parameters in the connection configuration file.")
```

Comment:
Raising RuntimeErrors is not a good practice. It is better to have some default parameters and use them in such cases, but warn the user about such problems in logs or by simple print.

Reply:
Resolved. Add default parameters. In case if this parameters incorrect it will be processed on the stage with connecting to DB.